extensions/interactions/GraphInput/directives/graph_viz_directive.html

```diff
@@ -19,7 +19,7 @@
   }
 </style>
 <div class="oppia-graph-input-viz-container">
-  <svg class="oppia-graph-viz-svg" width="100%" height="250" ng-mousemove="mousemoveGraphSVG($event)" ng-click="onClickGraphSVG($event)" viewBox="<[svgViewBox]>">
+  <svg class="oppia-graph-viz-svg" width="100%" height="250" ng-mousemove="mousemoveGraphSVG($event)" ng-click="onClickGraphSVG($event)"  ng-attr-view_box="<[svgViewBox]>">
```

Comment:
Shouldn't this be ng-attr-view-box?

Reply:
I don't think so as underscore here denotes that `viewBox` is actually camelcased. You may want to take a look at this [docs](https://docs.angularjs.org/guide/interpolation#-ngattr-for-binding-to-arbitrary-attributes) shared by @hoangviet1993.

Reply:
Oh I see, got it. Thanks!